### PR TITLE
add sign-in token api

### PR DESCRIPTION
**Pull request by PhilippBuschhaus**
Add the sign-in token api: 
https://clerk.com/docs/reference/backend-api/tag/Sign-in-Tokens

### CODE REVIEW

File: sign_in_token.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package clerk
+
+type SignInToken struct {
+	APIResource
+	Object    string `json:"object"`
+	ID        string `json:"id"`
+	Status    string `json:"status"`
+	UserID    string `json:"user_id"`
+	Token     string `json:"token,omitempty"`
+	URL       string `json:"url,omitempty"`
```
Comment (gkats) [inline sign_in_token.go]: I think the URL can also be `null`.

```suggestion
	URL       *string `json:"url,omitempty"`
```

File: signintokens/client.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
+// Package session provides the Sign In Token API.
+package signintokens
```
Comment (gkats) [inline signintokens/client.go]: :wrench: Let's change the package name (and the folder) to `signintoken`. Most of the package names in the library use singular form for the resources.

File: signintokens/client.go [hunk 3]
```diff
@@ -0,0 +1,51 @@
+// Package session provides the Sign In Token API.
```
Comment (gkats) [inline signintokens/client.go]:
```suggestion
// Package signintoken provides the Sign In Token API.
```

**Review by gkats — CHANGES_REQUESTED** [review]
Thank you so much for the contribution @PhilippBuschhaus!

Overall the code looks great. I've left a few comments, could you please take another look and then we can :shipit:.

**Comment by PhilippBuschhaus** [comment]
> Thank you so much for the contribution @PhilippBuschhaus!
> 
> Overall the code looks great. I've left a few comments, could you please take another look and then we can :shipit:.

Thanks for the feedback :). I've addressed the issues but let me know if anything else needs changing.

**Review by gkats — APPROVED** [review]

**Comment by MohamedArif20091999** [comment]
@PhilippBuschhaus I could see this code is merged, But With latest 	"github.com/clerk/clerk-sdk-go/v2" I'm not able to see this module, Is there something I'm missing here?

**Pull request merged**